custom/common/templates/common/drilldown_options.html

```diff
@@ -31,13 +31,11 @@
 </div>
 
 {% block filter_js %}
-<link href="{% static 'select2-3.4.5-legacy/select2.css' %}" rel="stylesheet"/>
-<script src="{% static 'select2-3.4.5-legacy/select2.min.js' %}"></script>
-
-<script type="text/javascript" src="{% static 'reports/ko/report_filter.drilldown_options.js' %}"></script>
-<script type="text/javascript" src="{% static 'common/ko/report_filter.drilldown_options.js' %}"></script>
 <script type="text/javascript">
-    $(function () {
+    $.when(
+        "{% static 'reports/ko/report_filter.drilldown_options.js' %}",
+        "{% static 'common/ko/report_filter.drilldown_options.js' %}",
```

Comment:
I didn't know you could use when like this directly on strings to mean "load these resources"